caffe2/core/operator.h

```diff
@@ -208,8 +208,12 @@ class CAFFE2_API OperatorBase : public Observable<OperatorBase> {
     Tensor tensor = caffe2::Tensor(output);
     if (!tensor.defined() || tensor.GetDeviceType() != type) {
       // Fix tensor type
-      tensor = Tensor(type);
-      output = at::Tensor(std::move(tensor.getIntrusivePtr()));
+      output = at::Tensor(
```

Comment:
why do you need to change this?

Reply:
because `getIntrusivePtr()` still returns the old `c10::intrusive_ptr<TensorImpl, UndefinedTensorImpl>` API isntead of the nullptr based `c10::intrusive_ptr<TensorImpl>`. This is going to be fixed later.